style.css

```diff
@@ -0,0 +1,107 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@media (min-width: 375px) {
+  head {
+    margin-top: 10px;
+  }
+
+  body {
+    padding: 1.5rem;
+    background-image: url("header-shapes mobile.png");
+    background-size: 100%;
+    background-position: cover;
+    background-repeat: no-repeat;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+
+  .menu a {
+    width: 20.27vw;
+    height: 5.33vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 4.8vw;
+    line-height: 74%;
+    color: #6070ff;
+    cursor: pointer;
+    text-decoration: none;
+  }
+
+  .menu {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .menu img {
+    width: auto;
+    height: auto;
+  }
+
+  .intro h1 {
+    font-family: 'Poppins', sans-serif;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    margin-bottom: 10px;
+    font-weight: 700;
+    color: #172b4d;
+  }
+
+  .intro p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5rem;
+    vertical-align: top;
+    color: #344563;
+  }
+
+  .connect h4 {
+    font-family: 'Poppins', sans-serif;
+    color: #7f8cff;
+    font-weight: 500;
+    font-size: 4.25vw;
+    margin: 0.9375rem 0;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 1rem;
+    height: 100vh;
+  }
+
+  .content {
+    margin-top: -9rem;
+  }
```

Comment:
- [ ] As requested by the previous CR, please 🙏🏽 correct the margins to be similar to the Figma template 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Porkalaikarthika/Portfolio/assets/58642949/37e5d5fe-0344-4190-8e82-212e94d60695) | ![image](https://github.com/Porkalaikarthika/Portfolio/assets/58642949/fc001063-b2fd-48a0-8ce1-16222e1b832c) |

Reply:
@franclobo 

![portfolioss](https://github.com/Porkalaikarthika/Portfolio/assets/130765186/8bea7117-db9a-418c-8c00-5f269e13ecee)
 kindly ensure is this the change to be made